diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 6fd5a1377f8d435aeb8a5e3effdb616090debc7a..3e9957a65313a0ed7beab843831d5f753b7becd5 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -274,7 +274,7 @@ void ToolbarView::Init() {
 
   std::unique_ptr<AvatarToolbarButton> avatar;
   if (show_avatar_toolbar_button)
-    avatar = std::make_unique<AvatarToolbarButton>(browser_);
+    avatar = std::make_unique<BraveAvatarToolbarButton>(browser_);
 
   auto app_menu_button = std::make_unique<BrowserAppMenuButton>(this);
   app_menu_button->EnableCanvasFlippingForRTLUI(true);
