diff --git a/chrome/browser/ui/cocoa/main_menu_builder.mm b/chrome/browser/ui/cocoa/main_menu_builder.mm
index 158d63677d0e6f4b81ae26c0b2fd69f31872857a..fdf9a9474d6883f43be8c01e400e8ca64df81bcc 100644
--- a/chrome/browser/ui/cocoa/main_menu_builder.mm
+++ b/chrome/browser/ui/cocoa/main_menu_builder.mm
@@ -97,6 +97,8 @@ base::scoped_nsobject<NSMenuItem> BuildFileMenu(
                 Item(IDS_NEW_INCOGNITO_WINDOW_MAC)
                     .command_id(IDC_NEW_INCOGNITO_WINDOW)
                     .remove_if(is_pwa),
+                Item(IDS_NEW_OFFTHERECORD_WINDOW_TOR)
+                    .command_id(IDC_NEW_OFFTHERECORD_WINDOW_TOR),
                 Item(IDS_REOPEN_CLOSED_TABS_MAC)
                     .command_id(IDC_RESTORE_TAB)
                     .remove_if(is_pwa),
@@ -418,6 +420,7 @@ base::scoped_nsobject<NSMenuItem> BuildHelpMenu(
 #if defined(GOOGLE_CHROME_BUILD)
             Item(IDS_FEEDBACK_MAC).command_id(IDC_FEEDBACK),
 #endif
+                Item(IDS_REPORT_BROKEN_SITE_MAC).command_id(IDC_SHOW_BRAVE_WEBCOMPAT_REPORTER),
                 Item(IDS_HELP_MAC)
                     .string_format_1(product_name)
                     .command_id(IDC_HELP_PAGE_VIA_MENU),
