diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 75e937301f223e61a5e902b6ef2a1f76ee0dea8f..406436131eeb0f9a047147d7f440cf3ec70fa6f8 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -721,6 +721,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome/browser/resources/media/mei_preload:component",
